src/main/java/svc/controllers/SMSController.java

```diff
@@ -0,0 +1,45 @@
+package svc.controllers;
+
+import java.io.IOException;
+import com.twilio.twiml.MessagingResponse;
+import com.twilio.twiml.TwiMLException;
+
+import javax.inject.Inject;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import svc.managers.*;
+import svc.models.*;
+
+@RestController
+@EnableAutoConfiguration
+@RequestMapping("/sms")
+public class SMSController {
+	@Inject
+	SMSManager smsManager;	
+		
+	@ResponseBody
+	@RequestMapping(method = RequestMethod.POST)
+	void GetMessage(@ModelAttribute TwimlMessageRequest twimlMessageRequest, HttpServletResponse response, HttpSession session) throws IOException{
```

Comment:
I'm wondering about this myself.  From their website, they say that you can use cookies or sessions.